openjdk8/src/main/docker/setup-env.d/20-debug-env.bash

```diff
@@ -8,5 +8,5 @@ fi
 
 if [ "$DBG_ENABLE" = "true" ]; then
   unset CDBG_DISABLE
-  DBG_AGENT="$( RUNTIME_DIR=$JETTY_BASE /opt/cdbg/format-env-appengine-vm.sh )"
+  DBG_AGENT="$(/opt/cdbg/format-env-appengine-vm.sh)"
```

Comment:
$ export DBG_AGENT="$(/usr/local/google/home/joaomartins/Downloads/cdbg_java_agent/format-env-appengine-vm.sh)"
$ echo $DBG_AGENT
-agentpath:/usr/local/google/home/joaomartins/Downloads/cdbg_java_agent/cdbg_java_agent.so=--log_dir=/var/log/app_engine,--logtostderr=false,--cdbg_extra_class_path=/webapps/root/WEB-INF/classes:/webapps/root/WEB-INF/lib

(without the format-env-appengine-vm.sh fix)